package.json

```diff
@@ -1,35 +1,32 @@
 {
-	"//1": "describes your app and its dependencies",
-	"//2": "https://docs.npmjs.com/files/package.json",
-	"//3": "updating this file will download and update your packages",
-	"name": "my-hyperdev-app",
+	"name": "boilerplate-project-messagebaord",
 	"version": "0.0.1",
-	"description": "What am I about?",
+	"description": "InfoSec project for freeCodeCamp",
 	"main": "server.js",
 	"scripts": {
 		"start": "node server.js"
 	},
 	"dependencies": {
-		"express": "^4.14.0",
-		"cors": "^2.8.1",
 		"body-parser": "^1.15.2",
 		"chai": "^3.5.0",
-		"mongodb": "^2.2.16",
 		"chai-http": "^3.0.0",
+		"cors": "^2.8.1",
+		"dotenv": "^8.2.0",
+		"express": "^4.14.0",
+		"helmet": "^3.1.0",
 		"mocha": "^3.2.0",
-		"zombie": "^5.0.5",
-		"helmet": "^3.1.0"
+		"mongodb": "^2.2.16",
+		"zombie": "^5.0.5"
```

Comment:
```suggestion
		"mongodb": "^3.5.8"
```

Reply:
Was there a decision or proposal on one of these PRs to remove the DB dependency entirely from the package.json?
Or am I mis-remembering?

Reply:
I believe it was just `mongoose`, but I could be wrong.

Reply:
I think you're right on that one. I don't see anywhere the connection is defined in the boilerplate - so I am going to say to yea, just remove it. Maybe @RandellDawson can give a quick second thought.